### ISSUE TITLE
Empty collection

### ISSUE THREAD

**Issue by AnyResidentOps** [crapola/blender_t3d#11]
I'm trying to import an old map made for Unreal Engine 2 / 2.5 in UnrealEd.

@crapola Do you know why importing the following map in Blender 4.0 results in an empty collection with no meshes?

https://gist.github.com/AnyResidentOps/1754e25777c75f5f8d1f3e170d62826c

**Comment by AnyResO** [comment]
Ah it seems your regex here expects classes to be only `Brush` while it could be `Engine.Brush` 

https://github.com/crapola/blender_t3d/blob/7f77cfd6c627c3aed6f9e6247813dc18f4270d5f/blender_t3d/t3d_parser.py#L30

see https://gist.github.com/AnyResO/1754e25777c75f5f8d1f3e170d62826c#file-mp-beml4-canyoncrusade-t3d-L32

**Comment by crapola** [comment]
Indeed, I think using `r"""(Begin Actor Class=(?:Engine.)?Brush .*?End Actor)"""` will work.
